### PR TITLE
docs: update delete command in UG

### PR DESCRIPTION
**Pull request by gigituang**

### CODE REVIEW

**Review by blaketingyu — APPROVED** [review]
LGTM fam. Think you can just merge. Pretty minor changes 🌮

File: docs/UserGuide.md [hunk 1]
```diff
@@ -151,12 +153,11 @@ Usage: (Automatically loads the recipes upon launching the application).
...
 | add a new recipe     | `add -n <name of recipe> -d <prep time of recipe> -ss <serving size/portions of recipe> -i <ingredients separated by commas> -s <steps separated by commas> [-t <tags separated by commas>]`                           |
 | edit a recipe        | `edit <name of recipe> -n <name of recipe> -d <prep time of recipe> -ss <serving size/portions of recipe> -i <ingredients separated by commas> -s <steps separated by commas> [-t <tags separated by commas>optional]` |
-| delete a recipe      | `delete <recipe number from list>`                                                                                                                                                                                     |
+| delete a recipe      | `delete -x <recipe number from list> or delete <recipe name>                                                                                                                                                                                    |
```
Comment (yusufaine) [inline docs/UserGuide.md:160]: is it possible to make the alternate command separated by a new line? perhaps something like:

```suggestion
| delete a recipe      | `delete -x <recipe number from list> <br> OR <br> delete <recipe name>                                                                                                                                                                                    |
```

**Review by yusufaine — APPROVED** [review]
Minor suggestion, but looks good. 🌮

File: docs/UserGuide.md (hunk 1 above)
Comment (blaketingyu) [reply docs/UserGuide.md:160]: Nice! 🔥

**Pull request merged**